src/chimera/seq.cljc

```diff
@@ -216,3 +216,17 @@
   (let [f (fn [[k v]] (if (keyword? k) [(subs (str k) 1) v] [k v]))]
     ;; only apply to maps
     (postwalk (fn [x] (if (map? x) (into {} (map f x)) x)) m)))
+
+(defn add-element
+  "Adds an element to a vector"
+  [vector element]
+  (-> vector
+      (conj (int element))
+      set
+      vec))
+
+(defn remove-element
+  "Removes an element to a vector"
+  [vector element]
+  (vec (remove (fn [id] (= id element))
```

Comment:
Feel like var name `id` misrepresents what the car actually us, maybe use `e`?

Reply:
Cool, that makes sense. Lemme change it.